lib/yardstick/decorator.rb

```diff
@@ -0,0 +1,60 @@
+module Yardstick
+  # A string decorator for applying unix console codes
+  class Decorator
+    FORMAT = "\e[%{mode};%{color}%{string}\e[0m".freeze
+
+    COLOR_CODES = {
+      red:    31,
+      yellow: 33
+    }.freeze
+
+    MODE_CODES = {
+      bold:      1,
+      underline: 4
+    }.freeze
+
+    private_constant(:FORMAT, :COLOR_CODES, :MODE_CODES)
```

Comment:
I used to have

here but this produced the following warning when generating YARD documentation:

```ruby
[error]: Unhandled exception in YARD::Handlers::Ruby::PrivateConstantHandler:
[error]:   in `lib/yardstick/decorator.rb`:16:

	16: private_constant(*constants(false))

[error]: NameError: uninitialized constant YARD::Handlers::Ruby::PrivateConstantHandler::NamespaceMissingError
[error]: Stack trace:
	/Users/johnbackus/.rvm/gems/ruby-2.2.3/gems/yard-0.8.7.6/lib/yard/handlers/ruby/private_constant_handler.rb:33:in `rescue in privatize_constant'
	/Users/johnbackus/.rvm/gems/ruby-2.2.3/gems/yard-0.8.7.6/lib/yard/handlers/ruby/private_constant_handler.rb:25:in `privatize_constant'
	/Users/johnbackus/.rvm/gems/ruby-2.2.3/gems/yard-0.8.7.6/lib/yard/handlers/ruby/private_constant_handler.rb:11:in `block (2 levels) in <class:PrivateConstantHandler>'
	/Users/johnbackus/.rvm/gems/ruby-2.2.3/gems/yard-0.8.7.6/lib/yard/handlers/ruby/private_constant_handler.rb:8:in `each'
	/Users/johnbackus/.rvm/gems/ruby-2.2.3/gems/yard-0.8.7.6/lib/yard/handlers/ruby/private_constant_handler.rb:8:in `block in <class:PrivateConstantHandler>'
	/Users/johnbackus/.rvm/gems/ruby-2.2.3/gems/yard-0.8.7.6/lib/yard/handlers/processor.rb:114:in `block (2 levels) in process'
```

since this is Yardstick this also happens when the specs are run which pollutes the rspec output.